rollup.config.js

```diff
@@ -0,0 +1,51 @@
+import resolve from '@rollup/plugin-node-resolve'
+import commonjs from '@rollup/plugin-commonjs'
+import typescript from '@rollup/plugin-typescript'
+import babel from '@rollup/plugin-babel'
+import { terser } from 'rollup-plugin-terser'
+import dts from 'rollup-plugin-dts'
+import del from 'rollup-plugin-delete'
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
+const packageJson = require('./package.json')
+
+const production = !process.env.ROLLUP_WATCH
+
+export default [
+  {
+    input: 'src/index.ts',
+    output: [
+      {
+        file: packageJson.main,
+        format: 'cjs',
+      },
+      {
+        file: packageJson.module,
+        format: 'esm',
+      },
+    ],
+    plugins: [
+      resolve({ moduleDirectories: ['ariakit', 'reakit'] }),
```

Comment:
just out of curiosity, why do we have only `ariakit` and `reakit` here?

Reply:
because they have specific imports like `ariakit/menu` and this a way of avoiding the ariakit menu component code be a part of the final bundle. The other way would be declaring every component of these dependencies in the external option.